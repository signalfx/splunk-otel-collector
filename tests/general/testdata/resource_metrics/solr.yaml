resource_metrics:
  - scope_metrics:
      - metrics:
          - name: counter.solr.http_2xx_responses
            type: IntMonotonicCumulativeSum
