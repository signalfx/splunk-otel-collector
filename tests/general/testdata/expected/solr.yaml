resourceMetrics:
  - resource: {}
    scopeMetrics:
      - metrics:
          - name: counter.solr.http_2xx_responses
            sum:
              aggregationTemporality: 2
              dataPoints:
                - asInt: "38"
                  attributes:
                    - key: cluster
                      value:
                        stringValue: default
                    - key: collection
                      value:
                        stringValue: techproducts
                    - key: core
                      value:
                        stringValue: techproducts_shard1_replica_n1
                    - key: dsname
                      value:
                        stringValue: value
                    - key: leader
                      value:
                        stringValue: "true"
                    - key: node
                      value:
                        stringValue: localhost:8983_solr
                    - key: plugin
                      value:
                        stringValue: solr
                    - key: plugin_instance
                      value:
                        stringValue: localhost:8983
                    - key: shard
                      value:
                        stringValue: shard1
                    - key: system.type
                      value:
                        stringValue: solr
                  timeUnixNano: "1000000"
              isMonotonic: true
        scope: {}
